governance.md

```diff
@@ -0,0 +1,145 @@
+
+- Have worked in a helpful and collaborative way with the MDN community.
+- Have given good feedback on others’ submissions and displayed an overall understanding of the code quality standards for the project.
+- Have the ability to drive the project forward, manage requirements from users, and taking on responsibility for the overall health of the project.
+
+An individual is invited to become an Owner by existing Owners. A nomination will result in discussion and then a decision by the Owner group.
+
+#### List of Owners
+- Florian Scholz (@Elchi3), Mozilla, BCD project lead
+- Will Bamberg (@wbamberg), Mozilla
+- Chris David Mills (@chrisdavidmills), Mozilla
+- Kadir Topal (@atopal), Mozilla
+
+The owners are also listed as top-level owners in GitHub’s code owner file.
+
+## Additional paths to becoming a Peer or Owner
+
+Some Owners or Peers are also [MDN Content Curators](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Documentation_topics_and_curators) and have thus earned the privilege to be a mdn-browser-compat-data Peer, so that their expertise in a given content area (CSS, HTML, etc.) can help improve the compat data for that same content area. Such Peers are marked in the relevant folders using GitHub’s Code Owner mechanism.
+
+Peers might also be representatives of browser vendors and have expertise and/or access to browser-specific information within their company. Their company name is listed in the Peer list.
+
+## Owner-delegate rule
+Owners may, at their discretion, nominate an owner-delegate to carry out a task or make a decision ordinarily carried out by an Owner. Delegation should be limited in duration or scope, or both; delegation may be withdrawn by any Owner at any time. For example, an Owner may nominate an owner-delegate to approve an infrastructure PR or to publish npm packages for a set time or number of releases.
+
+## Project Meetings
+There are no recurrent project meetings; they are scheduled when required at a time that works for the owners, and using tools that enable participation by the community. The meeting is run by a designated moderator approved by the owners group.
+Items are added to the agenda which are considered contentious or are modifications of governance, contribution policy, owner membership, or release process.
+
+The intention of the agenda is not to approve or review all patches. That should happen continuously on GitHub and be handled by the larger group of Peers. The exception to this is when defining how the schema should look (or when proposing an update), or when a PR discussion has stalled due to disagreement or inaction, and progress needs to be unblocked.
+
+Any community member or Peer can ask that something be added to the next meeting’s agenda by logging a GitHub Issue. Peers can add the item to the agenda by adding the “meeting agenda” tag to the issue and Contributors can ask Peers to add the label for them.
```

Comment:
"meeting agenda" should be a link to the tag, like [meeting-agenda](https://github.com/mdn/browser-compat-data/labels/meeting-agenda). I think you should go ahead and create the desired tag.